modules/ixBidAdapter.js

```diff
@@ -311,6 +313,19 @@ function getEidInfo(allEids) {
       }
     }
   }
+
+  const isValidFlocId = utils.deepAccess(flocData, 'id') && utils.deepAccess(flocData, 'version');
```

Comment:
deepAccess is a nice utility for deeply nested checks, but use on shallow objects like this are just going to contribute to slower evaluation. This does a string.split on every call[1]. Can't we just do `const isValidFlocId = flocData && flocData.id && flocData.version`?

[1] https://github.com/developit/dlv/blob/master/index.js

Reply:
Thank, yest that works for me.